cli/cli.go

```diff
@@ -0,0 +1,77 @@
+package cli
+
+import (
+	. "fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/codegangsta/cli"
+	"github.com/ugorji/go/codec"
+
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/executor"
+	"polydawn.net/repeatr/executor/null"
+)
+
+func GetApp() *cli.App {
+	app := cli.NewApp()
```

Comment:
This could just be a package var and the rest in an `init()` function.  We'll never want more than one of them.